src/apps/frontend/components/todo/TodoForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { createTodo } from '../../services/todo.service';
+type Props = {
+  onCreate: () => void;
+};
+
+const TodoForm: React.FC<Props> = ({ onCreate }) => {
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [type, setType] = useState('Personal');
+  const [dueDate, setDueDate] = useState('');
+
+  const handleSubmit = async () => {
+    await createTodo({
+      title,
+      description,
+      type: type as 'Personal' | 'Official' | 'Hobby',
+      due_date: `${dueDate}T00:00:00`,
```

Comment:
Why are we passing, we should be passing date simply.

This feels like we are band-aiding a solution